CHANGELOG.md

```diff
@@ -39,6 +39,12 @@ All notable changes to this project will be documented in this file.
 * ### NI-SCOPE
     * #### Added
     * #### Changed
+        * `niscope.fetch()` and `niscope.fetch_into()` now takes additional parameters for associated attributes and add resulting record as part of the waveform info
+        * Following properties are now private (added as parameters to fetch calls):
```

Comment:
Minor: The fact that they are private is an implementation detail. From the API point of view, the properties were removed.

`Following properties are removed. Use new parameters to `Session.fetch()` and `Session.fetch_into()`

Reply:
Changed